src/Contracts/MASA.Scheduler.Contracts.Server/Infrastructure/Utils/FluentValidationExtensions.cs

```diff
@@ -0,0 +1,104 @@
+﻿// Copyright (c) MASA Stack All rights reserved.
+// Licensed under the Apache License. See LICENSE.txt in the project root for license information.
+
+namespace Masa.Scheduler.Contracts.Server.Infrastructure.Utils;
+
+public static class FluentValidationExtensions
+{
+    public static IRuleBuilderOptions<T, string> Chinese<T>(this IRuleBuilder<T, string> ruleBuilder)
```

Comment:
Copy from mc?

Reply:
I will change it to use Masa.Utils.Extensions.Validations.FluentValidation when the library publish to nuget